src/sonic-yang-models/yang-models/sonic-mgmt_port.yang

```diff
@@ -0,0 +1,75 @@
+module sonic-mgmt_port {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-mgmt_port";
+    prefix mgmtprt;
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    description "MANAGEMENT PORT yang Module for SONiC OS";
+
+    revision 2021-04-07 {
+        description "First Revision";
+    }
+
+    container sonic-mgmt_port {
+
+        container MGMT_PORT {
+
+            description "MANAGEMENT PORT part of config_db.json";
+
+            list MGMT_PORT_LIST {
+
+                key "name";
+
+                leaf name {
+                    type string {
+                        pattern 'eth([1-3][0-9]{3}|[1-9][0-9]{2}|[1-9][0-9]|[0-9])';
+                    }
+                }
+
+                leaf speed {
+                    type uint16 {
+                        range "10|100|1000"; 
```

Comment:
By the way, from where we got this constraint on speed that it can take only these three values.

Reply:
These seem to be the supported speeds for mgmt interface currently